crates/astria-sequencer/src/app/mod.rs

```diff
@@ -892,7 +952,7 @@ impl App {
         proposer_address: account::Id,
         txs: Vec<bytes::Bytes>,
         tx_results: Vec<ExecTxResult>,
-    ) -> Result<()> {
+    ) -> Result<SequencerBlock> {
```

Comment:
I believe this would be faster since we do not have again read the `SequencerBlock` from state when we want to send it via the optimistic block channels.